model/model.go

```diff
@@ -309,7 +309,7 @@ func buildRouterConfig(rc *api.ReplicationController, platformCertSecret *api.Se
 		return nil, err
 	}
 	if platformCertSecret != nil {
-		platformCertificate, err := buildCertificate(platformCertSecret, "default")
+		platformCertificate, err := buildCertificate(platformCertSecret, "platform")
```

Comment:
The original text was wrong-- we stopped calling it the "default" cert a long time ago. (That actually means something else.) But, it's probable that should be a separate fix. I'll amend this and submit a separate PR for that. Thanks for keeping me honest.